modules/user_accounts/php/edit_user.class.inc

```diff
@@ -909,40 +909,68 @@ class Edit_User extends \NDB_Form
         // get the editor's permissions
         $perms = $editor->getPermissionsVerbose();
 
-        $lastRole = '';
-        $group    = [];
-        foreach ($perms as $row) {
-            if ($row['type'] != $lastRole) {
-                $lastRole = $row['type'];
-                $group[]  = $this->form->createElement(
-                    'static',
-                    '',
-                    '',
-                    '</div>'
-                    . "<h3 id=\"header_$lastRole\" "
-                    . "class=\"perm_header button\" "
-                    . "style=\"text-align: center; margin-top: 5px;\">"
-                    .ucwords($row['type'])
-                    . '</h3>'
-                    . "<div id=\"perms_$lastRole\" style=\"margin-top: 5px;\">"
-                );
+        usort(
```

Comment:
the sort could just as easily (if not easier be done on the query coming from SQL)